resources/views/news/index.blade.php

```diff
@@ -0,0 +1,30 @@
+@extends('layouts.app')
+
+@section('content')
+    <div>
+        <div>
+            <h1>News</h1>
+            @auth
+                <a href="{{ route('news.create') }}">Create new news</a>
+            @endauth
+            @if ($news->count())
+                @foreach ($news as $singleNews)
+                    <div>
+                        {{ $singleNews->title }}
+                        {!! \Illuminate\Mail\Markdown::parse($singleNews->content) !!}
```

Comment:
```suggestion
                        {!! Markdown::parse($singleNews->content) !!}
```
make alias for that.